content/deploy/install/keycloak.md

```diff
@@ -116,13 +122,42 @@ Create a client for the UI as follows:
        - 🗸 Direct access grants
        - 🗸 Implicit flow
 
+1. Configure the **Access Settings**:
+
+   - **Root URL**: `<UI_SUBDOMAIN>.<YOUR_DOMAIN>` without trailing slashes
+   - **Home URL**: `<UI_SUBDOMAIN>.<YOUR_DOMAIN>` without trailing slashes
+   - **Web Origins**: `<UI_SUBDOMAIN>.<YOUR_DOMAIN>` without trailing slashes
+
 1. Select **Next**, then **Save**.
 
-1. Scroll to **Access Settings** and configure:
+#### Create dashboard client
+
+1. In the side menu, select **Clients > create client**.
+1. Configure the **General Settings**:
 
-    - **Root URL**: The domain to host the {{< param brand_name >}} User Interface without trailing slashes
-    - **Home URL**: The domain to host the {{< param brand_name >}} User Interface without trailing slashes
-    - **Web Origins**: The domain to host the {{< param brand_name >}} User Interface without trailing slashes
+   - **Client Type**: `OpenID Connect`
+   - **Client ID**: `dashboard`
+   - **Name**: `{{< param brand_name >}} Dashboard`
+   - **Description**: `{{< param brand_name >}} Dashboard`
+
+1. Configure the **Capability config**:
+
+   - **Client Authentication**: On
+   - **Authorization**: On
+   - For **Authentication flow**, enable:
+     - 🗸 Standard flow
+     - 🗸 Direct access grants
+     - 🗸 Implicit flow
+
+1. Configure the **Access Settings**:
+
+   - **Root URL**: `<DASHBOARD_SUBDOMAIN>.<YOUR_DOMAIN>` without trailing slashes
+   - **Home URL**: `<DASHBOARD_SUBDOMAIN>.<YOUR_DOMAIN>` without trailing slashes
+   - **Valid redirect URIs**: `<DASHBOARD_URL>/login/generic_oath` without trailing slashes
```

Comment:
typo in `/generic_oauth`

Is there any difference between `<DASHBOARD_URL>` and `<DASHBOARD_SUBDOMAIN>.<YOUR_DOMAIN>` ?

I'd expect it to mirror  the others with this trailing path i.e. `<DASHBOARD_SUBDOMAIN>.<YOUR_DOMAIN>/login/generic_oauth`

Reply:
Good catches here, thanks!